universe/Conditions.cpp

```diff
@@ -1078,6 +1078,14 @@ unsigned int All::GetCheckSum() const {
 ///////////////////////////////////////////////////////////
 // None                                                  //
 ///////////////////////////////////////////////////////////
+None::None() :
+    Condition()
+{
+    m_root_candidate_invariant = true;
```

Comment:
same... should be all `false` I think? Doesn't matter what the root candidate, source, or target are; nothing will be matched.